evergreen.yml

```diff
@@ -696,7 +696,7 @@ functions:
           echo "------------------------------------"
 
           # Install CycloneDX CLI and JQ
-          JQ_URL="https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux-arm64"
+          JQ_URL="https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux-amd64"
           echo "-- Downloading JQ $JQ_URL --"
```

Comment:
I'm not sure why this was set to `arm64` when we were running it on `ubuntu2204-small`. In the JDBC driver and BIC, we're using `jq-linux-amd64`, so I changed it to that.

Reply:
I assume this was a typo. This is what irks me about how some institutions use amd64 for x86-64. Yes, I get, AMD created it as an extension to x86, but amd and arm are too similar and easy to mess up :joy: